cpp/src/reductions/scan/scan_inclusive.cu

```diff
@@ -62,26 +62,25 @@ rmm::device_buffer mask_scan(column_view const& input_view,
 namespace {
 
 /**
- * @brief Strings inclusive scan operator
+ * @brief Min/Max inclusive scan operator
+ *
+ * This operator will accept index values, check them and then
+ * run the `Op` operation on the individual element objects.
+ * The returned result is the appropriate index value.
  *
  * This was specifically created to workaround a thrust issue
  * https://github.com/NVIDIA/thrust/issues/1479
  * where invalid values are passed to the operator.
- *
- * This operator will accept index values, check them and then
- * run the `Op` operation on the individual string_view objects.
- * The returned result is the appropriate index value.
  */
-template <typename Op>
-struct string_scan_operator {
-  column_device_view const col;          ///< strings column device view
-  string_view const null_replacement{};  ///< value used when element is null
-  bool const has_nulls;                  ///< true if col has null elements
-
-  string_scan_operator(column_device_view const& col, bool has_nulls = true)
-    : col{col}, null_replacement{Op::template identity<string_view>()}, has_nulls{has_nulls}
+template <typename Element, typename Op>
+struct min_max_scan_operator {
```

Comment:
I wonder if `min_max` is too specific here.  The current use case is min/max but maybe down the road there's other operators that might make sense here.

Reply:
I will change the name when that happens.